core/runner/src/test/java/com/cognifide/aet/runner/processing/TimeoutWatchTest.java

```diff
@@ -1,3 +1,11 @@
+package com.cognifide.aet.runner.processing;
```

Comment:
We usually have license header at the top of our files. could we have it consistent here?